pulsar-broker/src/main/java/org/apache/pulsar/client/api/RawReader.java

```diff
@@ -30,9 +31,10 @@
     /**
      * Create a raw reader for a topic.
      */
-    public static CompletableFuture<RawReader> create(PulsarClient client, String topic) {
+    public static CompletableFuture<RawReader> create(PulsarClient client, String subscription,
```

Comment:
topic, subscription is a better sequence than subscription, topic.

Reply:
will change.